### PR TITLE
Friendship 2 - Next milestone

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
To build this part, we:

- use Rails 
- use git flow
- use rspec
- use linter configs

### CODE REVIEW

File: spec/friendship_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    click_link_or_button 'Sign in'
+    fill_in 'user[email]', with: user1.email
+    fill_in 'user[password]', with: '123456'
+    click_button 'Log in'
+    visit users_path
+    click_button 'Send request'
+    expect(page).to have_text('Friend request sent')
+    click_button 'Sign out'
+    fill_in 'user[email]', with: user2.email
+    fill_in 'user[password]', with: '123456'
```
Comment (DeliceLydia) [inline spec/friendship_spec.rb:67]: - Try to increase your units and integration tests to make your project less guilty 😊

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
### Hello @elyor-doniyorov! ✋😊

Incredible work so far, your social media app is working as expected you can send and receive friend requests.. however, you still need to make some changes to be able to go to the next milestone.

# Change Requested ♻️

Check comment under the review

## Cheers, and keep rocking! 💪💪

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. or you can reach out via Slack: Delice Lydia |  [Linkedin](linkedin.com/in/delice-lydia) | [Twitter](https://twitter.com/IngabireLydia3)

File: spec/friendship_spec.rb (hunk 1 above)
Comment (descholar-ceo) [reply spec/friendship_spec.rb:67]: This issue has been addressed successfully :heavy_check_mark:

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @elyor-doniyorov  , your project is approved :tada: :tada: :tada: 
You have implemented all requested requirements from the previous for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by elyor-doniyorov** [comment]
Thanks for the review!

**Pull request merged**